pytorch3dunet/bioimage-io/UNet3DArabidopsisOvules.model.yaml

```diff
@@ -0,0 +1,88 @@
+name: UNet3DArabidopsisOvules
+description: A 3d U-Net trained to predict the cell boundaries in confocal stacks of Arabidopsis ovules
+cite:
+  - text: "Wolny, Adrian et al. Accurate and Versatile 3D Segmentation of Plant Tissues at Cellular Resolution. BioRxiv 2020."
+    doi: https://doi.org/10.1101/2020.01.17.910562
+authors:
+  - Adrian Wolny;@bioimage-io
```

Comment:
We should refine the author string format, do we link user names or orgas? other tags?